tests/reports/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+include_directories(SYSTEM
+                    ${catch2_include_directory}
```

Comment:
Where are these defined?

Reply:
BTW, this is better IMO to just link against the Catch target, and it will take care of the include directories.

Reply:
Yes, I think the best idea is to have catch as a submodule and then just do
```
add_subdirectory(Catch2)
target_link_libraries(unittests PRIVATE Catch2::Catch2 ...)
```
As suggested in their documentation